std/node/util.ts

```diff
@@ -4,6 +4,25 @@ import * as types from "./_util/_util_types.ts";
 
 export { types };
 
+const DEFAULT_INSPECT_OPTIONS = {
+  showHidden: false,
+  depth: 2,
```

Comment:
Isn't the default `depth` 4?

Reply:
As I see in the related [docs](https://nodejs.org/dist/latest-v14.x/docs/api/util.html#util_util_inspect_object_options), default is 2 (since v11.4.0).